### DISCUSSION TITLE
Admin account

### DISCUSSION THREAD

**Discussion by pg0581** [sct/overseerr#1090] [Help] [answered]
When I set up Overseerr I used my Plex account which is a shared account. So I thought I would create a local admin account and then take the admin privileges away from the Plex account, but doesn't seem that this is possible?

**Comment by TheCatLady** [comment, marked as answer]
@pg0581 The account used to set up Overseerr is used to access your Plex server, so it is not possible to remove admin privileges or delete this "Owner" account.

That said, if you prefer to use local sign-in instead of Plex OAuth, you can simply add a password to this account.

**Reply by pg0581** [reply]
@TheCatLady I thought this was the case, bit I just wanted to ensure I wasn't missing something. Thanks for your prompt reply 😊

**Reply by enigmaoftech** [reply]
Is there a way we can have an option to have the owner/full admin as local user and then just have a setup option to link Plex like tautulli?

**Reply by kylerm42** [reply]
Agreed, this would be a great feature to have. My PMS was originally owned by a shared account, but I later realized that was a terrible idea, so I changed the owner. Now Overseerr can't connect to Plex at all because it's admin is the original account, so I'm having to choose between all my requests staying in "Requested" instead of "Available" or creating a new instance and losing all my history and requests for unreleased titles. Being able to change the Overseerr owner as well would be a massive help here.
